### PR TITLE
implement get_mapping

### PR DESCRIPTION
**Pull request by jeyeyeu**
Implemented two methods:
- `hanjp_keyboard_default_init(HanjpKeyboardInterface *iface)`
-  `gunichar hanjp_keyboard_get_mapping(HanjpKeyboard* self, gint tableid, gint ascii)` 
   +  calls `	hangul_keyboard_get_mapping` and returns the mapped character.

### CODE REVIEW

File: hanjpkeyboard.c [hunk 1]
```diff
@@ -5,7 +5,7 @@ G_DEFINE_INTERFACE(HanjpKeyboard, hanjp_keyboard, G_TYPE_OBJECT)
 
 static void
 hanjp_keyboard_default_init(HanjpKeyboardInterface *iface) {
-    /* add properties and signals to the interface here */
+	iface->get_mapping = hanjp_keyboard_get_mapping;
```
Comment (onting) [inline hanjpkeyboard.c:8]: Connecting the member functions with your implementation must be written in hanjp_keyboarddefault_keyboard_interface_init function, not here, and the function is already written as far as I remember.

**Review by onting — CHANGES_REQUESTED** [review]
Looks good to me,
without one point I mentioned.

File: hanjpkeyboard.c (hunk 1 above)
Comment (jeyeyeu) [reply hanjpkeyboard.c:8]: yeah it was implemented under `hanjp_keyboarddefault_init` which can be found on the bottom of the code. But as you can see from the comment - that I did not wrote - it was their to be implemented so I did it anyway.

Comment (onting) [reply hanjpkeyboard.c:8]: No, it's a function for signals and properties, not for initialization.
I fix it.

**Review by onting — APPROVED** [review]

**Pull request merged**